docs/computing/lumi-vs-mahti.md

```diff
@@ -0,0 +1,55 @@
+
+## Programming environment and software stacks
+
+The programming environment of LUMI-C is quite different compared to CSC supercomputers. LUMI comes with three alternative programming environments, namely Cray, GNU and AOCC. Each of the environments have their own compiler suites that become available upon loading the corresponding programming environment module. Moreover, two types of software stacks are offered, the CrayEnv and LUMI stacks. Please refer to the LUMI documentation for a detailed description of the [available collection of compiler suites](https://docs.lumi-supercomputer.eu/development/compiling/prgenv/) and [software stacks](https://docs.lumi-supercomputer.eu/computing/softwarestacks/) and how to swap between these.
+
+!!! Note
+    Irrespective of the loaded compiler suite, one noticeable difference concerning the LUMI-C programming environment is that it comes with compiler wrappers that replace commands commonly found on HPC systems such as Mahti. For example, commands for compiling MPI code like `mpicc`, `mpic++` and `mpif90` are unavailable as such. Instead, you should use the wrappers `cc`, `CC` and `ftn`, respectively. See the LUMI documentation for [more details on the available MPI wrappers](https://docs.lumi-supercomputer.eu/development/compiling/prgenv/#compile-an-mpi-program).
+
+## Disk areas
+
+Similar to CSC supercomputers, LUMI uses the Lustre parallel filesystem. However, there's currently no fast local disk such as on Puhti and Mahti-AI. The fast flash-based scratch space (LUMI-F) will be made available once the installation of LUMI-G is completed. Also, the object storage (LUMI-O) is not available yet. [See the LUMI documentation for more details](https://docs.lumi-supercomputer.eu/storage/).
+
+## Available partitions
+
+LUMI-C has two types of partitions (queues): two that are allocatable by node (only full nodes can be requested, similar to Mahti) and three that are allocatable by resources (partial nodes can be requested, similar to Puhti). [See more details in the LUMI documentation](https://docs.lumi-supercomputer.eu/computing/jobs/partitions/), e.g. maximum wall-time/node count and naming of the partitions.
+
+## Billing
+
+Billing on LUMI-C differs from Mahti, as well as depending on which partition you are using. Notably, the standard partition (allocatable by node) on LUMI-C is billed based on reserved CPU-core hours (128 billing units per node hour), while on Mahti reserving one node consumes 100 BUs per hour. The small partition on LUMI-C (allocatable by core) is on the other hand billed either 1) per reserved CPU-core (1 BU per CPUh), or 2) per allocated chunk of 2 GB of memory _if_ your are using more than 2 GB of memory per core. [See the LUMI documentation for more details and precise formulas](https://docs.lumi-supercomputer.eu/computing/jobs/billing/).
```

Comment:
I would not explain the model so much here, just say it is different and give link to billing.  A key difference that should be mentioned is that there are three currencies. GPU hours, cpu hours, and storage hours. This is a key difference to having only one BU